bgpd/bgp_damp.c

```diff
@@ -174,16 +174,9 @@ static void bgp_reuse_list_add(struct bgp_damp_info *bdi,
 }
 
 /* Delete BGP dampening information from reuse list.  */
-static void bgp_reuse_list_delete(struct bgp_damp_info *bdi,
-				  struct bgp_damp_config *bdc)
+static void bgp_reuse_list_delete(struct bgp_damp_info *bdi)
```

Comment:
Why not dropping this function at all?

Reply:
Could do that, just wanted to make a quick fix because it causes a crash. We can clean this up later.

Reply:
It works, I tested it.